bambi/interpret/utils.py

```diff
@@ -230,8 +104,17 @@ def get_model_terms(model: Model) -> dict:
 
 
 def get_model_covariates(model: Model) -> np.ndarray:
-    """
-    Return covariates specified in the model.
+    """Return covariates specified in the model.
```

Comment:
Reading through this function I'm wondering whether we're making it unnecessarily complicated by appending lists to `covariates` and then flattening it. Wouldn't it be more direct to just append the names themselves? I don't remember why we did things this way.